dev_requirements.txt

```diff
@@ -13,3 +13,5 @@ mypy==0.782
 wheel
 twine
 pytest-logbook>=1.2.0,<1.3
+git+https://github.com/fishtown-analytics/dbt-mashumaro.git@dbt-customizations
```

Comment:
We will want to move this into setup.py before distrubuting obviously, I'm happy to follow up after the PR with how we should package this.

Reply:
Yes. We'll also have to do a new release of Hologram, which I added to dev_requirements.txt for now.

Reply:
^ This is the comment I'm still tracking. It's not possible to install from `setup.py` on this branch. We need an answer here before we can put this out as a beta version, which I'd like to do as soon as possible. Could one of you open a follow-on issue?

Unless I'm mistaken, it also means we can't tell people to test this out (after merging) via:
```
pip install git+https://github.com/fishtown-analytics/dbt.git
```